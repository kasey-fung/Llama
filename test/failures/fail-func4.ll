void printTwo(int a, int b):
    print(a)
    print(b)

void main():
    printTwo(1, 2, 3)
