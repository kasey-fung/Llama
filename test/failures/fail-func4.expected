Fatal error: exception Failure("expecting 3 arguments in printTwo(1, 2)")